lib/harness/renderer/run.ex

```diff
@@ -52,17 +52,26 @@ defmodule Harness.Renderer.Run do
       |> Enum.map(&Renderer.File.source(&1, run, File.lstat!(&1).type))
       |> Enum.reject(&(&1.output_path == Path.join(".harness", "pkg.exs")))
 
-    links =
+    symlinks =
       run.generator_config
       |> run.generator_module.links()
       |> Enum.map(&Renderer.File.source(&1, run, :symlink))
 
+    hard_links =
+      run.generator_config
+      |> run.generator_module.hard_links()
+      |> Enum.map(&Renderer.File.source(&1, run, :hard_link))
```

Comment:
the idea with the optional callbacks being:

```suggestion
    hard_links =
      if function_exported?(run.generator_module, :hard_links, 1) do
        run.generator_config
        |> run.generator_module.hard_links()
        |> Enum.map(&Renderer.File.source(&1, run, :hard_link))
      else
        []
      end
```